mobilecoind/src/service.rs

```diff
@@ -214,10 +214,7 @@ impl<T: BlockchainConnection + UserTxConnection + 'static, FPR: FogPubkeyResolve
     ) -> Result<mc_mobilecoind_api::AddMonitorResponse, RpcStatus> {
         // Get the AccountKey from the GRPC request.
         let proto_account_key = request.account_key.as_ref().ok_or_else(|| {
-            RpcStatus::new(
-                RpcStatusCode::INVALID_ARGUMENT,
-                Some("account_key".to_string()),
-            )
+            RpcStatus::with_message(RpcStatusCode::INVALID_ARGUMENT, "account_key".to_string())
```

Comment:
Can these all be `"foo".into()` to match the stuff in `consensus/service/src/api/grpc_error.rs` above?